modules/prebidServerBidAdapter/index.js

```diff
@@ -395,13 +397,97 @@ const OPEN_RTB_PROTOCOL = {
           return { w, h };
         });
 
-        banner = {format};
+        mediaTypes['banner'] = {format};
       }
 
-      let video;
       const videoParams = utils.deepAccess(adUnit, 'mediaTypes.video');
       if (!utils.isEmpty(videoParams)) {
-        video = videoParams;
+        mediaTypes['video'] = videoParams;
+      }
+
+      const nativeParams = processNativeAdUnitParams(utils.deepAccess(adUnit, 'mediaTypes.native'));
+      if (nativeParams) {
+        let assetCounter = 200;
+        try {
+          mediaTypes['native'] = {
+            request: JSON.stringify({
+              // TODO: determine if following OpenRTB recommended properties can be defined or are needed
+              // context: int,
+              // plcmttype: int,
+              // privacy: int
```

Comment:
I'm not sure if any of the above _recommended_ properties in the OpenRTB spec need to somehow be added to the `adUnit.mediaTypes.native` properties?